integration-tests/integration-tests/src/test/java/dev/rico/integrationtests/AbstractIntegrationTest.java

```diff
@@ -61,7 +68,7 @@ public AbstractIntegrationTest() {
         try {
             final URL dockerComposeURL = AbstractIntegrationTest.class.getClassLoader().getResource("docker-compose.yml");
             final Path dockerComposeFile = Paths.get(dockerComposeURL.toURI());
-            final BackgroundExecutor backgroundExecutor = Client.getService(BackgroundExecutor.class);
+            final Executor backgroundExecutor = Executors.newCachedThreadPool(new SimpleThreadFactory());
```

Comment:
I do not know if change is correct. We should discuss this next week.

Reply:
This is only in a test and the executor is only used to startup the docker containers